### DISCUSSION TITLE
Why do you select the Basque word `berri` to name this package?

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [wannier-berri/wannier-berri#109] [Q&A] [answered]
Today, I read your npj paper on this project [here](https://www.nature.com/articles/s41524-021-00498-5.pdf). You give the following explanation for the project name:

```
The name of the code is derived from Wannier functions and the Basque word “berri”which means “new”and enters local
toponyms (e.g., Lekunberri, Ekainberri).
```
But I really didn't know about the distant Basque until I read your paper. Why do you select  the Basque word `berri` to name this package?

Regards,
HZ

**Comment by stepan-tsirkin** [comment, marked as answer]
@hongyi-zhao  thank you for the question.  Maybe it was a goal that people who never heard about Basque Country get to know about it. 

One may also notice that the logo is written in a traditional Basque font, and the colors are similar to the flag of the region.

But basically the name is because "berri" sounds exactly like "Berry" (phase/curvature) which is a central quantity for the code. 

And it is a nice place where I spent some years of my life. If you work on smth related to WannierBerri code, maybe you will find interesting to attend some of the events that a held in San Sebastian, e.g.  the yearly [Topological Matter School](https://tms-dipc.org/) .

**Reply by hongyi-zhao** [reply]
> One may also notice that the logo is written in a traditional Basque font, and the colors are similar to the flag of the region.

Do you mean [this one](http://wannier-berri.org/_images/WANNIERBERRI-line-redblack.png)?

> But basically the name is because "berri" sounds exactly like "Berry" (phase/curvature) which is a central quantity for the code.

This is also the first impression emerged in my mind when I saw this word as part of the project name.

> And it is a nice place where I spent some years of my life. 

I guessed the possibility, too. But as a very unknown country, its level of education and research should not be world-famous. At least not at the same level  as the University of Zurich, where you work now.

> If you work on smth related to WannierBerri code, maybe you will find interesting to attend some of the events that a held in San Sebastian, 

I get an explanation about the relationship between San Sebastián and Basque from [here](https://groups.google.com/g/alt.usage.english/c/V_9MDBHg_3Y/m/ihIzrWfdBQAJ) and [here](https://en.wikipedia.org/wiki/San_Sebasti%C3%A1n):

```
San Sebastian, also known as Donostia-San Sebastián (official names in both local languages: Donostia (Basque pronunciation: [doˈnos̺tia]) and San Sebastián (Spanish: [s̺an s̺eβas̺ˈtjan]))[4] is a city and municipality located in the Basque Autonomous Community, Spain.
```

> e.g. the yearly [Topological Matter School](https://tms-dipc.org/) .

Thank you for telling me this. I'm very interested in the Wannier related method currently and found the following two invaluable places on the website you mentioned above:

https://twitter.com/TopologicalS
https://www.youtube.com/channel/UCXlN-CTCH5j4Kiec6YlSf9w/featured

**Reply by stepan-tsirkin** [reply]
>  But as a very unknown country, its level of education and research should not be world-famous. At least not at the same level as the University of Zurich, where you work now.

Well, officially it is not even a country, but an autonomous region within Spain.  Although such renown physicist like Einstein or Pauli did not work in the Basque Country, you may be surprised how high is the level of science in San Sebastian and Bilbao, in particular in the fields of topological materials and [crystallography](https://www.cryst.ehu.es/)

**Reply by hongyi-zhao** [reply]
Thank you for the further additions and explanations. I knew about Bilbao Crystallographic Server a long time ago, but don't realize that it also related to autonomous Basque Country till now. The following explanation is given [here](https://en.wikipedia.org/wiki/Bilbao):

```
Bilbao (/bɪlˈbaʊ, -ˈbɑːoʊ/, also US: /-ˈbeɪoʊ/,[3][4][5] Spanish: [bilˈβao]; Basque: 
Bilbo [bilβo]) is a city in northern Spain, the largest city in the province of Biscay 
and in the Basque Country as a whole.
```
